### PR TITLE
Change Malay language code (#1109)

### LINKED ISSUE #1109: Change Malay language code to "ms"
There is a mistake in the code it was chosen, it happens that "my" is the country code for Malaysia, but it is the language code for the Burmese language.
"ms" is the code for Malay.

Related:
- https://github.com/doomspork/elixir-school/issues/1090
- https://github.com/doomspork/elixir-school/pull/1106

Reference: 
https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes

### PR DESCRIPTION
**Pull request by mbirman**
Hi,

This PR fixes #1109. Updates:
1) `index.md` files
2) `_data/interlang.yml`
3) `README.md`

And of course I renamed `my` directory to `ms`.

Please review it and let me know any feedback. Thanks.

### CODE REVIEW

**Comment by eksperimental** [comment]
If you ise the redirect from, you wont need to keep the my directory (which btw doesnt appear in the commit because git doesn't care about empty dirs)

**Comment by mbirman** [comment]
@eksperimental thanks, I updated the PR as requested.

File: ms/lessons/basics/sigils.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ---
 version: 0.9.0
 title: Sigil
+redirect_from:
+  - /my/lessons/basics/sigils/
```
Comment (doomspork) [inline ms/lessons/basics/sigils.md]: Thanks for addressing the redirects 👍

**Comment by doomspork** [comment]
Thank you so much @mbirman and @eksperimental 👍

**Pull request merged**